views/question-table.pug

```diff
@@ -1,7 +1,8 @@
 table.table
   thead
     tr
-      th Visible
+      th
+        button(id='vis' type='button' value='true' onClick='toggleButtonVisibility()') Visiblity
```

Comment:
can we use a button from the material design library

Reply:
I changed it to a switch from the material design library